Update insert-functions.md (#5224)
segmentio/segment-docs: src/connections/functions/insert-functions.md

Insert functions can define handlers for each message type in the [Segment spec](/docs/connections/spec/):

> info "onBatch handler"

> > Destination Insert Functions do not support onBatch handler yet, and Segment is actively working on this functionality. 

Reviewer: ```suggestion
> At this time, Destination Insert Functions do not support the onBatch handler. 
```
